packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -521,3 +521,38 @@ class BackdropFilterLayer extends ContainerLayer {
     builder.pop();
   }
 }
+
+/// A physical model layer.
+class PhysicalModelLayer extends ContainerLayer {
+  /// Creates a layer with a rounded-rectangular clip.
+  ///
+  /// The [clipRRect] property must be non-null before the compositing phase of
+  /// the pipeline.
+  PhysicalModelLayer({ this.clipRRect, this.elevation, this.color });
```

Comment:
Maybe add `@required` to clipRRect and assert that it's non-null?  Are any of the other parameters required to be non-null?